ql/src/test/queries/clientpositive/values.q

```diff
@@ -0,0 +1,10 @@
+set hive.cli.print.header=true;
+
```

Comment:
Can we add `explain cbo` for some of these queries?

Reply:
added